9.mAP/notebooks/1.aggregate_map_scores_class_level.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>It seems that the p-value pattern of <code>0.5 and 1.0</code> is still here. If this  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>